src/engine/configuration.cpp

```diff
@@ -581,4 +560,22 @@ Configuration loadConfigurationFromFile(const std::filesystem::path& configurati
     return result;
 }
 
+openspace::Configuration::LayerServer stringToLayerServer(std::string_view server) {
+    using Server = openspace::Configuration::LayerServer;
+    if (server == "All") { return Server::All; }
+    else if (server == "NewYork") { return Server::NewYork; }
+    else if (server == "Sweden") { return Server::Sweden; }
+    else if (server == "Utah") { return Server::Utah; }
+}
```

Comment:
```suggestion
    throw ghoul::MissingCaseException();
}
```
or else a warning is raised here because of a missing return value